RAQUASI88/src/keyboard.h

```diff
@@ -173,10 +173,11 @@ enum {
 
   KEY88_INVALID     = 0,
 
-  /* 1〜31 は特殊機能制御用にリサーブしておく */
+  /* 1〜30 は特殊機能制御用にリサーブしておく */
```

Comment:
if 31 was reserved before, is it okay to repurpose it?

Reply:
That's the point of reserved identifiers, isn't it?

Reply:
0-31 have special non-key meanings in ASCII (9 is tab, 10 is newline, etc). 31 is "unit separator", and I'm not sure if I've ever seen it use.

Reply:
OK, I don't really agree with the layout but it would make more sense to move it to the tail end, I'll change it with the rest.

Reply:
I searched some more and it turns out that it is `KEY88_UNDERSCORE` (already defined). On the keyboard it does not look like one so I thought they were distinct. It is also mappable only manually. I will roll back the `KEY88_DASH` addition.